pmd-java/src/main/java/net/sourceforge/pmd/lang/java/types/internal/infer/ResolutionFailedException.java

```diff
@@ -27,6 +28,7 @@
 
     private static final ThreadLocal<ResolutionFailedException> SHARED = ThreadLocal.withInitial(ResolutionFailedException::new);
     private static final boolean SHARE_EXCEPTION = true;
+    private static final TypePrettyPrinter PRETTY_PRINTER_CONFIG = new TypePrettyPrinter().qualifyTvars(true);
```

Comment:
This might be dangerous to store in a static field, since the TypePrettyPrinter is not threadsafe.

Reply:
I'll convert this to a local variable where we need it when merging.